### PR TITLE
Add Makefile for the running commands

### PR DESCRIPTION
**Pull request by Xennis**
Do you like that idea or not?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Set the GCP project ID: 
+# export GCP_PROJECT=<project-id>
+GCP_REGION = europe-west1
+GCP_BUCKET = dataflow-training-persistent
+GCP_TEMP_BUCKET = dataflow-training-temp
+
+directLocal:
```
Comment (carsten-luckmann) [inline Makefile]: I would add a prefix `run-` to the targets which run a pipeline

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		--staging_location gs://$(GCP_TEMP_BUCKET)/customer/staging \
+		--temp_location gs://$(GCP_TEMP_BUCKET)/temp \
+		--job_name customer-orders \
+		--setup_file ./setup.py \
+		--detail_input gs://$(GCP_TEMP_BUCKET)/customer/testdata/detail.json \
+		--order_input gs://$(GCP_TEMP_BUCKET)/customer/testdata/order.json \
+		--output gs://$(GCP_TEMP_BUCKET)/customer/output/output \
+		--output_aggregation avg
+
+createTemplate:
```
Comment (carsten-luckmann) [inline Makefile]: I think `make template` is sufficient.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		--temp_location gs://$(GCP_TEMP_BUCKET)/temp \
+		--template_location gs://$(GCP_BUCKET)/templates/customer \
+		--setup_file ./setup.py \
+		--detail_input gs://$(GCP_TEMP_BUCKET)/customer/testdata/detail.json \
+		--order_input gs://$(GCP_TEMP_BUCKET)/customer/testdata/order.json \
+		--output gs://$(GCP_TEMP_BUCKET)/customer/output/output
+	@echo "Upload the metadata file of the pipeline next to the template"
+	gsutil cp customer_metadata gs://$(GCP_TEMP_BUCKET)/templates/
+
+runTemplate:
```
Comment (carsten-luckmann) [inline Makefile]: Like here, just that I think `run-` is more idiomatic for Makefiles.

**Review by carsten-luckmann — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Xennis) [reply Makefile]: Good idea

File: Makefile (hunk 2 above)
Comment (Xennis) [reply Makefile]: Right

File: Makefile (hunk 3 above)
Comment (Xennis) [reply Makefile]: Okay

**Pull request merged**